### ISSUE TITLE
Can't load module myTesla

### ISSUE THREAD

**Issue by powergrim** [xbeaudouin/domoticz-tesla-plugin#1]
I`ve installed the plugin on Domoticz using Python 3.7.3. I`ve also installed myTesla and I`m able to import it manually. But the domoticz log still tells me that the module myTesla can't be found.

Log:

020-01-02 13:57:57.669 Status: (Tesla) Started.
2020-01-02 13:57:57.799 Status: (Tesla) Entering work loop.
2020-01-02 13:57:57.800 Status: (Tesla) Initialized version 0.1.0, author 'Xavier Beaudouin'
2020-01-02 13:57:57.803 Error: (Tesla) onStart: Domoticz Python env error myTesla import error: No module named 'myTesla'
2020-01-02 13:58:07.822 (Tesla) onHeartbeat called

**Comment by xbeaudouin** [comment]
Hi !
Currently the module is under construction (eg not working yet). Your need to install python module "myTesla" :https://github.com/zmsp/python-my-tesla 
Notice this module is under construction. So don't expect it work now :)

**Comment by powergrim** [comment]
I`ve installed the myTesla module manually and I called it using the python console. That seems to work fine. It's just that the Tesla Domoticz plugin can't detect the myTesla module.

I`ve got interaction with my Tesla working with Powershell scripts running Powershell Core now. So I`ve got a decent work-around that is working. But it would be nice to get this working since Python comes natively. Anything I can do to help?

**Comment by xbeaudouin** [comment]
Well does the python version on your console is the same as the one you have on Domoticz ? It should work.
Any help is appreciated. 
Current issue I have is to make this skeleton working (eg : keep the configuration work)... Unfortunatly kids take me too much time (didn't had a minute at home this xmas holidays).

**Comment by powergrim** [comment]
Yep, both are 3.7.3. the plugin system loads with the same error.

The thing I have using Powershell is to keep the connection awake. Might be the same issue you have. I`m not a Python adept, but I do get the API and the inner workings now. So exchange of thought might help.

**Comment by xbeaudouin** [comment]
Strange... Well you know this is my firt domoticz plugin... So I discover how this works.
What things you need and currently added with your powershells scripts ?

**Comment by xbeaudouin** [comment]
Hello, I have some stuff that is seem to work (more or less). Nothing on the Domoticz UI, but... on the log... (you should have vehicules informations).

**Comment by powergrim** [comment]
I`ve created a repository here:
https://github.com/powergrim/Tesla_Powershell_Script.git

That repo holds the Powershell script that controls my Tesla. I`m able to call it via Powershell core on a RasPi. I`m just controlling conditioning, charge state etc.. How did you get it to work? Using the Python Console? Or are you calling a py-script to get it running?
